src/OrbitGl/CoreMath.h

```diff
@@ -21,9 +21,4 @@ using Vec2i = gte::Vector2<int>;
 using Vec3i = gte::Vector3<int>;
 using Vec4i = gte::Vector4<int>;
 
-using Color = gte::Vector4<unsigned char>;
-
-template <class T>
-constexpr const T& clamp(const T& v, const T& lo, const T& hi) {
-  return std::min(std::max(v, lo), hi);
-}
+using Color = gte::Vector4<unsigned char>;
```

Comment:
seems like this file is missing include guard?

Reply:
Hmm yes. You're right. I didn't even notice that. I will add them.

Reply:
Ah. It has `pragma once`. I will change it to include guards

Reply:
done.